db/migrations/20240930100434_add_snap_uuid_to_templates.up.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+ALTER TABLE templates_repository_configurations ADD COLUMN IF NOT EXISTS snapshot_uuid UUID DEFAULT NULL;
```

Comment:
This question kinda goes to the question below about the ramifications around a null value here.  

Would it be possible to (eventually) have the column be not null?  (it might take more than 1 PR to pull this off).

Reply:
good point! i made the default `snapshot_uuid` value null because in the API we allow template creation with repositories without snapshots. if we wanted to make this not null, we'd have to validate that all repositories have snapshots before they're added to a template i think

Reply:
I think we'd also need to migrate any existing snapshots that are in templates to be included on the table